Testreglar/3.3.1/Nett/nett-3.3.1b.json

```diff
@@ -0,0 +1,256 @@
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "2.4"
+		},
+		{
+			"title": "2.5"
+		},
+		{
+			"title": "2.6"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har testsiden digitale skjema/skjemaelement?",
+			"ht": "<p><strong>Merk: </strong>Du skal ikke teste skjema i PDF, Word eller lignende.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden har ikke skjema/skjemaelement."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Hvilket skjema/skjemaelement tester du?",
+			"ht": "<p>Beskriv skjema/skjemaelement, slik at det er mulig å identifisere det i ettertid. </p>\n<p><strong>Merk: </strong>Hvis skjema/skjemaelement inngår i en prosess skal du også beskrive prosessen.</p>",
+			"type": "tekst",
+			"label": "Skjema/prosess:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Er det forhåndsutfylt informasjon i skjema?",
+			"ht": "<p>Forhåndsutfylt informasjon betyr at det allerede står inndata i skjemaet når testsiden er lastet inn.  </p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.5"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "2.6"
+				}
+			}
+		},
+		{
+			"stegnr": "2.5",
+			"spm": "Er det mulig å endre forhåndsutfylt informasjon i et eller flere skjemaelementer til feil format eller verdi?",
+			"ht": "<p>Sjekk om du kan endre forhåndsutfylt informasjon i et eller flere skjemaelementer til feil format eller verdi.</p>\n<p>For eksempel endre forhåndsutfylt informasjon i et skjemaelement for navn fra bokstaver til tall, eller omvendt for et skjemaelement som krever telefonnummer eller dato.</p>\n<p><strong>Merk:</strong> Hvis du ikke får endre informasjonen i minst et av skjemaelementene som er forhåndsutfylt, skal skjemaet ikke testes.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.6"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden krever ikke inndata i et bestemt format eller innenfor bestemte verdier."
```

Comment:
Usikker på logikken her. Hvis man svarer nei på spørsmålet sier man at det ikke er mulig å endre skjemaelement til feil format eller verdi. Altså krever skjemaet bestemt format eller verdi? for meg høres det ut som utfallet skulle vært "Testsiden krever inndata i et bestemt format eller innenfor bestemte verdier"

Reply:
Fjernet dette steget slik at testregel 3.3.1b er i samsvar med endringer i 3.3.1a